nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -143,6 +144,8 @@ BtNavigator::loadBehaviorTree(const std::string & bt_xml_filename)
   if (current_bt_xml_filename_ == bt_xml_filename) {
     return true;
   }
+  // if a new tree is created, than the ZMQ Publisher must be destroyed
+  bt_->resetZMQGrootMonitor();
```

Comment:
If this is a groot monitor - I don't think the ZMQ details here are relevant. Rename these functions to remove ZMQ.

Reply:
done